src/react/account/AccountUserSecretKeyModal.tsx

```diff
@@ -130,8 +144,14 @@ const modalBody = (key: AccountKey | null) => {
           </T.Row>
         </T.Body>
       </Table>
+      <Box display="flex" alignItems="flex-end" flexDirection="column">
+        <StyledCopybutton
+          text={`Account Name\t${key.accountName}\nUsername\t${IAMUserName}\nAccess key ID\t${key.accessKey}\nSecret Access key\t${key.secretKey}`}
```

Comment:
Following the previous comment, key.accountName is not really Account Name but User Name.